ext/spl/spl_observer.c

```diff
@@ -946,7 +946,7 @@ ZEND_END_ARG_INFO();
 
 ZEND_BEGIN_ARG_INFO_EX(arginfo_attach, 0, 0, 1)
 	ZEND_ARG_INFO(0, object)
-	ZEND_ARG_INFO(0, inf)
+	ZEND_ARG_INFO(0, data)
```

Comment:
Should update https://github.com/php/php-src/blob/0fe767d559cb52f38433e0e1c15c9f299058cca8/ext/spl/spl_observer.c#L396.